src/templates/joomlade/css/editor.css

```diff
@@ -24,6 +24,24 @@
 
 /*!
  * joomlade v0.0.1 (http://www.joomla.de)
+ * Copyright 2015-2017 Joomla.de Team
+ * Licensed under GPL v3 (http://www.gnu.org/licenses/gpl-3.0.de.html)
+ */
+
+/*!
+ * joomlade v0.0.1 (http://www.joomla.de)
```

Comment:
in der Datei steht nun 4 mal das gleiche untereinander ;)
![image](https://cloud.githubusercontent.com/assets/2596554/23654780/a187604e-0332-11e7-929f-97e219659926.png)